tests/common/mod.rs

```diff
@@ -130,6 +150,13 @@ where
     Ok(())
 }
 
+impl Drop for GitRepo {
+    fn drop(&mut self) {
+        fs::remove_dir_all(self.repo.path())
+            .expect("unable to clean up test repo");
+    }
+}
+
```

Comment:
Oops. Approved too soon. This is actually not necessary as when `TempDir` is dropped, it will delete the temp directory.